R/dash.R

```diff
@@ -1893,15 +1897,15 @@ Dash <- R6::R6Class(
       if ("/favicon.ico" %in% names(private$asset_map$other)) {
         favicon <- sprintf("<link href=\"/_favicon.ico\" rel=\"icon\" type=\"image/x-icon\">")
       } else {
-        favicon <- ""
+        favicon_url <- sprintf('\"%s_favicon.ico\"', self$config$requests_pathname_prefix)
```

Comment:
Have we tested this in a deployed application? Might not hurt to do so before merging the PR, something simple and quick on `dash-playground` should do (since we use `requests_pathname_prefix` here).

Reply:
I ended up testing it on `dash-core-docs`, and although there was a bit of a snag with the requests, I made a slight change and it's working as intended now both locally and deployed on DE.